keras_preprocessing/image/utils.py

```diff
@@ -127,10 +127,21 @@ def load_img(path, grayscale=False, color_mode='rgb', target_size=None,
     return img
 
 
-def list_pictures(directory, ext='jpg|jpeg|bmp|png|ppm'):
+def list_pictures(directory, ext=('jpg', 'jpeg', 'bmp', 'png', 'ppm')):
+    """Lists all pictures in a directory, including all subdirectories.
+
+    # Arguments
+        directory: string, absolute path to the directory
+        ext: tuple of strings, extensions of the pictures, separated by '|'
+    # Returns
+        a list of paths
+    """
+    if not isinstance(ext, tuple):
+        ext = (ext,)
+    ext = tuple('.%s' % e for e in ext)
```

Comment:
better to check if string
`ext = tuple('.%s' % e for e in ((ext,) if isinstance(ext, str) else ext))`